static/app/views/eventsV2/queryList.tsx

```diff
@@ -208,6 +225,21 @@ class QueryList extends React.Component<Props> {
           )}
           renderContextMenu={() => (
             <ContextMenu>
+              <Feature
+                organization={organization}
+                features={['connect-discover-and-dashboards']}
+              >
+                {({hasFeature}) =>
+                  hasFeature && (
+                    <MenuItem
```

Comment:
the prebuilt queries should also have the option to add to dashboard (since you can add them from the query detail page)

Reply:
Updated! 👍